WordPress/Classes/Services/AppUpdate/AppStoreSearchService.swift

```diff
@@ -0,0 +1,50 @@
+import Foundation
+
+struct AppStoreLookupResponse: Decodable {
+    let results: [AppStoreInfo]
+}
+
+struct AppStoreInfo: Decodable {
```

Comment:
(nit) I would suggest nesting it.

Reply:
Done! [4b0e962](https://github.com/wordpress-mobile/WordPress-iOS/pull/23195/commits/4b0e962b4a6152bd50564a262f809f8d484de30f)